server/protocols/webadmin/webadmin-core/src/main/java/org/apache/james/webadmin/WebAdminServer.java

```diff
@@ -131,20 +131,20 @@ private void configureExceptionHanding() {
             .statusCode(NOT_FOUND_404)
             .type(NOT_FOUND)
             .message(String.format("%s %s can not be found", req.requestMethod(), req.pathInfo()))
-            .toResponse(res));
+            .asString(res));
 
         service.internalServerError((req, res) -> ErrorResponder.builder()
```

Comment:
It would more be:

```
service.internalServerError((req, res) -> {
    res.status(INTERNAL_SERVER_ERROR_500);
    res.body(ErrorResponder.builder()
                    .statusCode(INTERNAL_SERVER_ERROR_500)
                    .type(SERVER_ERROR)
                    .message("WebAdmin encountered an unexpected internal error")
                   .asString()) );
    return res;
});
```